tests/Unit/inc/ThirdParty/Plugins/SimpleCustomCss/updateCacheFile.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\ThirdParty\Plugins\SimpleCustomCss;
+
+use Brain\Monkey\Functions;
+use WP_Rocket\Tests\Unit\FilesystemTestCase;
+use WP_Rocket\ThirdParty\Plugins\SimpleCustomCss;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Plugins\SimpleCustomCss::update_cache_file
+ * @group  ThirdParty
+ * @group  WithSCCSS
+ */
+class Test_DeleteCacheFile extends FilesystemTestCase {
+	protected $path_to_test_data = '/inc/ThirdParty/Plugins/SimpleCustomCss/updateCacheFile.php';
+	private $sccss;
+
+	public function setUp() {
+		parent::setUp();
+		$this->sccss = new SimpleCustomCss();
+
+		Functions\expect( 'rocket_get_constant' )->with( 'WP_ROCKET_CACHE_BUSTING_PATH' )->andReturn( 'wp-content/cache/busting/' );
+		Functions\expect( 'rocket_get_constant' )->with( 'WP_ROCKET_CACHE_BUSTING_URL' )->andReturn( 'http://example.org/wp-content/cache/busting/' );
```

Comment:
Here too, you can written this like this if you want:
```suggestion
		Functions\expect( 'rocket_get_constant' )
		    ->with( 'WP_ROCKET_CACHE_BUSTING_PATH' )
		    ->andReturn( 'wp-content/cache/busting/' )
		    ->andAlsoExpectIt()
		    ->with( 'WP_ROCKET_CACHE_BUSTING_URL' )
		    ->andReturn( 'http://example.org/wp-content/cache/busting/' );
```